### PR TITLE
join_questions_answers

### PR DESCRIPTION
**Pull request by jjessieshang**
Separated the single previous load stored procedure into two procedures:
1. load_questions (loads question and question type)
2. join_questions_answers (uses inner join to load questions, question type, and answers associated by foreign key)

### CODE REVIEW

**Review by Harin329 — CHANGES_REQUESTED** [review]
Sorry, I think I should've clarified this better. So ideally we want:

load_questions: To load all the questions, join them with the answers table and costs table. This will be our master list and will be how the customer facing form is populated. Your prev implementation is good, but you just need to select * (select everything) instead of question, answer and cost. Basically we want all the columns from all 3 tables.

join_questions_answers: Since you started this one, maybe we can use this to populate the editor form (since the cost wont be necessary for that and we can make the response payload lighter). Something like:
- QID1, What service do you want?, multi, position0, AnswerID1, Cutting, answerPosition0
- QID1, What service do you want?, multi, position0, AnswerID2, Slicing, answerPosition1
- QID1, What service do you want?, multi, position0, AnswerID3, Preparing, answerPosition2

**Comment by jjessieshang** [comment]
Ok makes sense, I've made those changes to load_questions in the latest commit, and have kept the lighter join_questions_answers the same.

File: database/join_questions_answes.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
+USE `labby.db`;
+DROP procedure IF EXISTS `join_questions_answers`;
+
+DELIMITER $$
+ 
+CREATE PROCEDURE `join_questions_answers` ()
```
Comment (Harin329) [inline database/join_questions_answes.sql:6]: Awesome thanks! Also one small thing, can we name this something and the `drop procedure` above something else so the function call name is more descriptive? Like: 

```suggestion
CREATE PROCEDURE `load_questions_answers` ()
```

File: database/join_questions_answes.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
+USE `labby.db`;
+DROP procedure IF EXISTS `join_questions_answers`;
```
Comment (Harin329) [inline database/join_questions_answes.sql:2]:
```suggestion
DROP procedure IF EXISTS `load_questions_answers`;
```

**Review by Harin329 — CHANGES_REQUESTED** [review]

File: database/load_questions_answes.sql [hunk 3]
```diff
@@ -0,0 +1,21 @@
+USE `labby.db`;
```
Comment (Harin329) [inline database/load_questions_answes.sql:1]: Nice! Sorry, one last thing I think this file is spelled wrong, maybe we can correct the `answes` :)

Comment (jjessieshang) [reply database/load_questions_answes.sql]: oh no so sorry!

**Review by Harin329 — APPROVED** [review]

**Pull request merged**